app/editor/src/features/admin/ingests/IngestList.tsx

```diff
@@ -41,28 +44,23 @@ const IngestList: React.FC<IIngestListProps> = (props) => {
       </Row>
       <IngestFilter
         onFilterChange={(filter) => {
-          if (filter && filter.length) {
-            const value = filter.toLocaleLowerCase();
-            setItems(
-              ingests.filter(
-                (i) =>
-                  i.name.toLocaleLowerCase().includes(value) ||
-                  i.source?.code.toLocaleLowerCase().includes(value) ||
-                  i.description.toLocaleLowerCase().includes(value) ||
-                  i.ingestType?.name.toLocaleLowerCase().includes(value),
-              ),
-            );
-          } else {
-            setItems(ingests);
-          }
+          setFilter(filter);
         }}
       />
       <FlexboxTable
         rowId="id"
         data={items}
         columns={columns}
         showSort={true}
-        onRowClick={(row) => navigate(`${row.original.id}`)}
+        // onRowClick={(row) => navigate(`${row.original.id}`)}
+        onCellClick={async (cell, row) => {
+          if (cell.index !== 6) navigate(`${row.original.id}`);
```

Comment:
I was thinking something similar.  I'd need to update the FlexboxTable component to provide a way to map to a `uid` or `key` rather than the index.  But it'll have to wait for another free minute or ten.